flamingpy/utils/viz.py

```diff
@@ -412,27 +405,31 @@ def draw_EGraph_plotly(
         hoverinfo="none",
     )
 
+    # axis
     axis = dict(
-        showbackground=True,
+        showbackground=kwargs.get("showbackground", False),
         showline=True,
+        showspikes=kwargs.get("showspikes", False),
         zeroline=False,
-        showgrid=True,
+        showgrid=kwargs.get("showgrid", True),
         showticklabels=show_axes,
         tickmode="linear",
         tick0=0,
         dtick=1,
     )
 
+    # layout
     layout = go.Layout(
-        width=750,
-        height=750,
+        width=kwargs.get("width", 750),
+        height=kwargs.get("height", 750),
         showlegend=legend,
```

Comment:
I don't think the plotly legend behaves the same way as the matplotlib legend right now (i.e. explains what the node colours mean). Should we leave this for a future PR?

Reply:
> Should we leave this for a future PR?

Yes, I tried out some stuff today but it seems like it's not trivial. I hoped it was similar to the colours/hover info where you can simply pass an array but that's not the case for the labels that show up in the legend.

Reply:
One solution would be to plot separate "traces" for each category, but this seems relatively complex. I'd propose to leave it for a future PR.